examples/Example/src/rows/index.tsx

```diff
@@ -1,22 +1,43 @@
+/* eslint-disable react/no-unescaped-entities */
```

Comment:
You could maybe change it as in the other files, @jakub-gonet ?

Reply:
Yep, I saw that but didn't add a comment. We should remove as much Eslint and ts ignores from examples as we can.

Reply:
Ok, so I leave it up to you to remove it.

Reply:
Fixed in d77e96d.